garage/tf/policies/gaussian_lstm_policy_with_model.py

```diff
@@ -0,0 +1,295 @@
+            of recurrent layer(s). The function should return a
+            tf.Tensor.
+        output_nonlinearity (callable): Activation function for output dense
+            layer. It should return a tf.Tensor. Set it to None to
+            maintain a linear activation.
+        output_w_init (callable): Initializer function for the weight
+            of output dense layer(s). The function should return a
+            tf.Tensor.
+        output_b_init (callable): Initializer function for the bias
+            of output dense layer(s). The function should return a
+            tf.Tensor.
+        hidden_state_init (callable): Initializer function for the
+            initial hidden state. The functino should return a tf.Tensor.
+        hidden_state_init_trainable (bool): Bool for whether the initial
+            hidden state is trainable.
+        cell_state_init (callable): Initializer function for the
+            initial cell state. The functino should return a tf.Tensor.
+        cell_state_init_trainable (bool): Bool for whether the initial
+            cell state is trainable.
+        forget_bias (bool): If True, add 1 to the bias of the forget gate at
+            initialization. It's used to reduce the scale of forgetting at the
+            beginning of the training.
+        learn_std (bool): Is std trainable.
+        std_share_network (bool): Boolean for whether mean and std share
+            the same network.
+        init_std (float): Initial value for std.
+        layer_normalization (bool): Bool for using layer normalization or not.
+        state_include_action (bool): Whether the state includes action.
+            If True, input dimension will be
+            (observation dimension + action dimension).
+    """
+
+    def __init__(self,
+                 env_spec,
+                 hidden_dim=32,
+                 name='GaussianLSTMPolicyWithModel',
+                 hidden_nonlinearity=tf.nn.tanh,
+                 hidden_w_init=tf.glorot_uniform_initializer(),
+                 hidden_b_init=tf.zeros_initializer(),
+                 recurrent_nonlinearity=tf.nn.sigmoid,
+                 recurrent_w_init=tf.glorot_uniform_initializer(),
+                 output_nonlinearity=None,
+                 output_w_init=tf.glorot_uniform_initializer(),
+                 output_b_init=tf.zeros_initializer(),
+                 hidden_state_init=tf.zeros_initializer(),
+                 hidden_state_init_trainable=False,
+                 cell_state_init=tf.zeros_initializer(),
+                 cell_state_init_trainable=False,
+                 forget_bias=True,
+                 learn_std=True,
+                 std_share_network=False,
+                 init_std=1.0,
+                 layer_normalization=False,
+                 state_include_action=True):
+        if not isinstance(env_spec.action_space, Box):
+            raise ValueError('GaussianLSTMPolicy only works'
```

Comment:
maybe also print the user what they passed, to help them understand the difference

Reply:
Should we print the class? like
```python
            raise ValueError('GaussianLSTMPolicy only works with '
                             'akro.tf.Box action space, but not {}'
                             .format(type(env_spec.action_space)))
```

Reply:
i think printing just `env_spec.action_space` should produce a nice output, no?

Reply:
It gives us Discrete(2) while type gives us the full class name, i.e. akro.tf.Discrete

Reply:
ok lgtm